src/base/bittorrent/session.cpp

```diff
@@ -4909,6 +4909,7 @@ void Session::handleSessionStatsAlert(const lt::session_stats_alert *p)
 
     m_status.hasIncomingConnections = static_cast<bool>(stats[m_metricIndices.net.hasIncomingConnections]);
 
+    // XXX: all of these should be std::int64_t once libtorrent <= 1.1.x is no longer supported
```

Comment:
Not sure of the difference but I think TODO is better.

Reply:
I only used `XXX` because it is used in many other places in the code base, so I thought it was the standard. But sure, I'll change to TODO.

Reply:
fixed